src/monitoring/mod.rs

```diff
@@ -104,6 +106,28 @@ pub fn increment_btc_blocks_received_counter() {
     prometheus::BTC_BLOCKS_RECEIVED_COUNTER.inc();
 }
 
+/// Log `execution_cost` as a ratio of `block_limit`.
+#[allow(unused_variables)]
+pub fn set_last_execution_cost_observed(
+    execution_cost: &ExecutionCost,
+    block_limit: &ExecutionCost,
+) {
+    #[cfg(feature = "monitoring_prom")]
```

Comment:
Is there a way to just group all of these statements inside a single `#[cfg(feature = "monitoring_prom")]`?  Something like:

```
#[cfg(feature = "monitoring_prom")] {
   /* do stuff here
}
```

Reply:
did this